util/src/main/java/io/camunda/zeebe/util/sched/ActorThread.java

```diff
@@ -74,19 +74,14 @@ private void doWork() {
       timerJobQueue.processExpiredTimers(clock);
     }
 
-    currentTask = taskScheduler.getNextTask(clock);
+    currentTask = taskScheduler.getNextTask();
 
     if (currentTask != null) {
-      try {
-        final var actorName = currentTask.actor.getName();
-        try (final var timer = actorMetrics.startExecutionTimer(actorName)) {
-          executeCurrentTask();
-        }
-        actorMetrics.countExecution(actorName);
-
-      } finally {
-        taskScheduler.onTaskReleased(currentTask);
```

Comment:
:question: What does this do before? :thinking:

Reply:
Nothing! :smile: There was only an empty default implementation in `TaskScheduler`

Reply:
:see_no_evil: